sktalk/corpus/conversation.py

```diff
@@ -61,11 +62,53 @@ def metadata(self):
         """
         return self._metadata
 
+    @property
+    def n_utterances(self):
+        """
+        Get the number of utterances in the conversation.
+
+        Returns:
+            int: The number of utterances in the conversation.
+        """
+        return len(self._utterances)
```

Comment:
Now you can do `len(conversation)` and get the number of utterances in it:

```suggestion
    def __len__(self):
        """
        Get the number of utterances in the conversation.

        Returns:
            int: The number of utterances in the conversation.
        """
        return len(self._utterances)
```

Reply:
Requires updating the tests.